docs/copy-repo.rst

```diff
@@ -0,0 +1,31 @@
+copy-repo
+=========
+
+.. argparse::
+   :module: pubtools._pulp.tasks.copy_repo
+   :func: doc_parser
+   :prog: pubtools-pulp-copy-repo
+
+
+Examples
+........
+
+Copy a repository from one location to another:
```

Comment:
It's not really copying a repository from one location to another.

This seems more accurate:

```
Copy content from one repository to another:
```